runtime/src/accounts_index.rs

```diff
@@ -1747,7 +1747,8 @@ impl<T: IndexValue> AccountsIndex<T> {
     /// on return, the index's previous account info may be returned in 'reclaims' depending on 'previous_slot_entry_was_cached'
     pub fn upsert(
         &self,
-        slot: Slot,
+        new_slot: Slot,
+        _old_slot: Slot,
```

Comment:
Can you add doc comments on what `old_slot` is? This can be on the next PR too, just wanted to call it out.